apps/web2/src/pages/EventPlanPage/EventPlanning.tsx

```diff
@@ -130,15 +130,16 @@ export default function EventPlanning({
 
   return (
     <Container fluid id="eventPlanningContainer">
-      <h1>{eventPlanData.name}</h1>
+      <br></br>
+      <h1 className="font-medium text-5xl mt-0 mb-2 text-black-600">{eventPlanData.name}</h1>
       <Col id="containerCol" sm={6}>
```

Comment:
I copied and pasted that from the tailwind site

Reply:
Container and Col are from bootstrap

Reply:
I had gotten the template from an online forum that used tailwind, but they probably added some bootstrap in there too because certain aspects of tailwind look very jumbled. It's a lot cleaner to put "<Container>" and "<Col>" rather than "<div class="max-w-7xl mx-auto sm:px-6 lg:px-8">". I think in this case we can just leave it as it is.

Reply:
Perhaps the best solution here is to create our own generic `Container` component here. Something like:

```
function Container(props) {
  return <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">{props.children}</div>
}
```

Doing something like this makes it easy to read while also standardizing the use of those tailwind css classes.